src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
```

Comment:
Delete comments after the task is completed, this is a good practice especially in a production environment (unless it says otherwise in the task content)